.github/workflows/updatecli.yml

```diff
@@ -14,13 +14,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
-        env:
-          BRANCH_NAME: ${{ github.ref_name }}
         with:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
-          pipeline: ./.ci/update-specs.yml
+          pipeline: .ci/updatecli.d
```

Comment:
I've been thinking about this particular pattern, using a folder rather than a file name, and even it's a bit more code (when using single pipelines), I think it's easier to debug when things fail.

I cannot find what's the failure about when running several pipelines sequentially, for instance see https://github.com/elastic/apm-agent-java/actions/runs/4486889582/jobs/7889783446

```
Run Summary
===========
Pipeline(s) run:
  * Changed:	0
  * Failed:	1
  * Skipped:	0
  * Succeeded:	2
  * Total:	3
ERROR: ✗ 1 over 3 pipeline failed
ERROR: command failed
```

What do you think?

Reply:
Is this still an issue? I see you somehow resolved this problem here https://github.com/elastic/apm-agent-java/actions/workflows/updatecli.yml

Anything you want to share here?

Reply:
This comment was the very first one before I debugged what caused the error, so we can use this approach, just wanted to share that using folders rather than individual pipelines had this pitfall for debugging what failed.

My other comments contain a suggestion to fix the potential error if no `title` in the pipeline name

Reply:
So, if the suggestions are applied.. it's easier to debug? Or do you think, we should go back to not using folders?

Reply:
BTW, one of the suggestions fixes the below error:

```
	action stage:	"Title is too long (maximum is 256 characters)"
```